test/test_settings.py

```diff
@@ -350,3 +350,120 @@ def test_cannot_set_invalid_values_for_max_header_list_size(self, val):
 
             with pytest.raises(KeyError):
                 s[h2.settings.MAX_HEADER_LIST_SIZE]
+
+class EqualityTestsMixin(object):
+    """
+    A mixin defining tests for the standard implementation of C{==} and C{!=}.
```

Comment:
Please also replace all instances of `C{}` surrounding text: we don't use that style in this project.